Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -58,7 +58,7 @@ std::vector<std::shared_ptr<MutableValue>> extractMutablesFromArray(jsi::Runtime
 
 NativeReanimatedModule::NativeReanimatedModule(std::shared_ptr<CallInvoker> jsInvoker,
                                                std::shared_ptr<Scheduler> scheduler,
-                                               std::unique_ptr<jsi::Runtime> rt,
+                                               std::unique_ptr<jsi::Runtime>& rt,
```

Comment:
this does not seem like a good idea, if the point of the pointer is to be referenced in a number of places we should use shared_ptr and not unique_ptr -> unique is for usecases where there is only one "holder"